.github/workflows/ci.yaml

```
@@ -71,7 +71,7 @@ jobs:
           fi   
 
       - name: Upload SARIF file
-        uses: github/codeql-action/upload-sarif@v2
+        uses: github/codeql-action/upload-sarif@v3.26.9@461ef6c76dfe95d5c364de2f431ddbd31a417628
```

```suggestion
        uses: github/codeql-action/upload-sarif@461ef6c76dfe95d5c364de2f431ddbd31a417628 # v3.26.9
```
For consistency reasons I'd prefer if we could keep the same format for this update.